pkgs/by-name/gr/gradia/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  appstream,
+  desktop-file-utils,
+  gobject-introspection,
+  wrapGAppsHook4,
+  blueprint-compiler,
+  libadwaita,
+  libportal-gtk4,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "gradia";
+  version = "1.4.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "AlexanderVanhee";
+    repo = "Gradia";
+    tag = "v${version}";
+    hash = "sha256-9iXz4MjgG4VlystZZIKOheK4hYuz6lRXPivy+1K+cYs=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    appstream
+    desktop-file-utils
+    gobject-introspection
+    wrapGAppsHook4
+    blueprint-compiler
+  ];
+
+  buildInputs = [
+    libadwaita
+    libportal-gtk4
+  ];
+
+  dependencies = with python3Packages; [
+    pygobject3
+    pillow
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}") # prevent double wrapping
+  '';
```

Nit:
```suggestion
  makeWrapperArgs = [ "\${gappsWrapperArgs[@]}" ];
```
Should probably use the list syntax here